### ISSUE TITLE
Empty

### ISSUE THREAD

**Issue by jkomyno** [AslayRey/cats#1]
Lol Bares ma crei repository vuote a caso?

**Comment by AslayRey** [comment]
No, in realtà penso "Dai, oggi mi metto a fare qualcosa", ma poi non faccio niente lol
